docs/GettingStarted/configuration/ExporterWebhook.md

```diff
@@ -49,8 +49,16 @@ This is the list of options that can be applied to `env_from_secrets`, `env_from
 
 | Variable | Required | Default Value |
 |----------|----------|---------------|
+| [SECRET_TOKEN](#secret_token) | no | - |
 | [LOG_LEVEL](#log_level) | no | `INFO` |
 
+###### SECRET_TOKEN
+
+- **Required:** no
+- **Type:** string
```

Comment:
Could not be `bool`? I only saw it being used in code as `if secret`. I feel `bool` is easier to type than `None | str`

Reply:
No, the secret is string value and it needs to be specified by the user.

Reply:
where this is used in the code?

Reply:
When you start app it's visible as REDACTED:

```
04-05-2023 14:13:58 INFO     Registered webhook plugin for user-agent: 'pelorus-webhook/'
04-05-2023 14:13:58 INFO     Loading WebhookCollector, inputs below:
04-05-2023 14:13:58 INFO     app_label='app.kubernetes.io/name', default value; APP_LABEL was not set
04-05-2023 14:13:58 INFO     secret_token=REDACTED, from env var SECRET_TOKEN
```

When unset:
```
04-05-2023 15:04:56 INFO     Registered webhook plugin for user-agent: 'pelorus-webhook/'
04-05-2023 15:04:56 INFO     Loading WebhookCollector, inputs below:
04-05-2023 15:04:56 INFO     app_label='app.kubernetes.io/name', default value; APP_LABEL was not set
04-05-2023 15:04:56 INFO     secret_token=REDACTED, default value; SECRET_TOKEN was not set
```

Reply:
This has nothing to do with the authentication TOKEN.

This is a value used to sign the payload. See similarly how github webhook is securing payload, we do quite same thing, except our payload must be signed by the sender and in the github example it's the github service that is signing it before sending.

https://docs.github.com/en/webhooks-and-events/webhooks/securing-your-webhooks#setting-your-secret-token

It is being used in the code see a new method in the app.py:
```
def _get_hash_token() -> str:
    return collector.secret_token
```

This method is used in the line
```
    handler = webhook_handler(request.headers, request, secret=_get_hash_token())
```

It needs to be a method that takes a value from a collector object, otherwise it's really hard to write test cases that mocks entire collector which is an async func.

Reply:
As we discussed, it's value is being used, so everything is ok